src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/sealedclaim/SealedClaimResponseForm.java

```diff
@@ -0,0 +1,172 @@
+            .responseType(caseData.getRespondent1ClaimResponseTypeForSpec())
+            .mediation(caseData.getResponseClaimMediationSpecRequired() == YesOrNo.YES)
+            .build();
+    }
+
+    private static void addRepaymentMethod(CaseData caseData, SealedClaimResponseForm.SealedClaimResponseFormBuilder builder, BigDecimal totalAmount) {
+        if (caseData.isPayImmediately()) {
+            addPayByDatePayImmediately(builder, totalAmount);
+        } else if (caseData.isPayByInstallment()) {
+            addRepaymentPlan(caseData, builder, totalAmount);
+        } else if (caseData.isPayBySetDate()) {
+            addPayBySetDate(caseData, builder, totalAmount);
+        }
+    }
+
+    private static void addPayBySetDate(CaseData caseData, SealedClaimResponseForm.SealedClaimResponseFormBuilder builder, BigDecimal totalClaimAmount) {
+        builder.payBy(caseData.getRespondToClaimAdmitPartLRspec().getWhenWillThisAmountBePaid())
+            .amountToPay(totalClaimAmount + "")
+            .whyNotPayImmediately(caseData.getResponseToClaimAdmitPartWhyNotPayLRspec());
+    }
+
+    private static void addPayByDatePayImmediately(SealedClaimResponseForm.SealedClaimResponseFormBuilder builder, BigDecimal totalClaimAmount) {
+        builder.payBy(LocalDate.now()
+                          .plusDays(RespondentResponsePartAdmissionPaymentTimeLRspec.DAYS_TO_PAY_IMMEDIATELY))
+            .amountToPay(totalClaimAmount + "");
+    }
+
+    private static void addRepaymentPlan(CaseData caseData, SealedClaimResponseForm.SealedClaimResponseFormBuilder builder, BigDecimal totalClaimAmount) {
+        if (caseData.getRespondent1RepaymentPlan() != null) {
+            builder.repaymentPlan(RepaymentPlanTemplateData.builder()
+                                      .paymentFrequencyDisplay(caseData.getRespondent1RepaymentPlan().getPaymentFrequencyDisplay())
+                                      .firstRepaymentDate(caseData.getRespondent1RepaymentPlan().getFirstRepaymentDate())
+                                      .paymentAmount(MonetaryConversions.penniesToPounds(caseData.getRespondent1RepaymentPlan().getPaymentAmount()))
+                                      .build())
+                .payBy(caseData.getRespondent1RepaymentPlan()
+                           .finalPaymentBy(totalClaimAmount))
+                .whyNotPayImmediately(caseData.getResponseToClaimAdmitPartWhyNotPayLRspec());
+        }
+    }
+
+    private static void alreadyPaid(CaseData caseData, SealedClaimResponseForm.SealedClaimResponseFormBuilder builder) {
+        RespondToClaim respondToClaim = caseData.getResponseToClaim();
+        builder.whyReject("ALREADY_PAID")
+            .howMuchWasPaid(MonetaryConversions.penniesToPounds(respondToClaim.getHowMuchWasPaid()) + "")
+            .paymentDate(respondToClaim.getWhenWasThisAmountPaid())
+            .paymentHow(respondToClaim.getHowWasThisAmountPaid() == PaymentMethod.OTHER
```

Comment:
PLUS for ternary operator, but too long for an argument. Put in separate var

Reply:
I agree there should not be a plus there. the code was borrowed from another dev that wrote it originally and i missed this. It should be toString() there